src/exes/moc.ml

```diff
@@ -191,7 +191,7 @@ let process_files files : unit =
   | Check ->
     Diag.run (Pipeline.check_files files)
   | Viper ->
-    let s = Diag.run (Pipeline.viper_files files) in
+    let s = Diag.run (fst (Pipeline.viper_files files)) in
```

Comment:
```suggestion
    let (s, _) = Diag.run (Pipeline.viper_files files) in
```
Seems to be required otherwise I get a type error build moc

Reply:
Thanks! Looks much clearer this way as well.